src/models/sighting/useSightingTermQuery.js

```diff
@@ -3,14 +3,20 @@ import { getSightingTermQueryKey } from '../../constants/queryKeys';
 
 export default function useSightingTermQuery(searchTerm) {
   const query = {
-    query_string: {
-      query: `*${searchTerm}*`,
-      fields: [
-        'guid',
-        'verbatimLocality',
-        'owners.full_name',
-        'comments',
```

Comment:
correct. it will be in advanced search but i realized sighting quicksearch doesn't advertise searching on comments and it would probably slow things down